### PR TITLE
Add SystemInfo page

### PR DESCRIPTION
**Pull request by maggie0002**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix
[X] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
Added a page that calls systeminfo to fetch various device details. 

**Which issue (if any) does this pull request address?**

**Is there anything you'd like reviewers to focus on?**
Note it is based off the other pull request. That one should be merged before this one.

### CODE REVIEW

**Review by nucleardreamer — APPROVED** [review]
hey this is great! we should probably test on the fleet before merging, but it looks great from code review perspective.

**Comment by nucleardreamer** [comment]
I pinned the test device to this PR build, looks good so far!!

**Comment by maggie0002** [comment]
> I pinned the test device to this PR build, looks good so far!!

I know there will be certain features that do not work, but seems like it just returns an empty JSON object in those instances. I thought about testing them, but some of the ones that aren't working could potentially work depending on setup. For example if someone mounts the Balena engine UNIX socket to the container, or uses Bluetooth on their deice. Seemed to be too many eventualities to predict them all, so just added in what there was. Well, actually I just copied the code from the tests that Systeminfo maintainers had built, and did a find and replace to format it, so not particularly thorough, but hopefully useful.

**Pull request merged**
